cfgov/apache/conf.d/redirects.conf

```
@@ -1026,3 +1026,9 @@ RewriteRule ^(?!.*(csv|html|json|jsonp|xml)$).*$ https://ffiec.cfpb.gov/data-pub
 ## Except for requests to specific file endpoints, which we mark as gone
 RewriteCond %{HTTP_HOST} api.consumerfinance.gov
 RewriteRule ^.+(csv|html|json|jsonp|xml)$ - [G,NC]
+ 
+ 
+# Strip trailing period for domain name
+# https://stackoverflow.com/questions/31972945/how-to-remove-the-trailing-dot-after-tld-by-using-mod-rewrite
+RewriteCond %{HTTP_HOST} ^(.+?)\.$
+RewriteRule ^ http://%1%{REQUEST_URI} [L,R=302,NE]
```

```suggestion
RewriteRule ^ http://%1%{REQUEST_URI} [L,R=301]
```

I think we want these to be permanent redirects, and I'm pretty sure we don't need the No Escape flag.